runtime-light/stdlib/server/http-functions.cpp

```diff
@@ -181,3 +181,16 @@ void f$setrawcookie(const string &name, const string &value, int64_t expire_or_o
   }
   kphp::http::header({static_SB_spare.c_str(), static_SB_spare.size()}, false, kphp::http::status::NO_STATUS);
 }
+
+array<string> f$headers_list() noexcept {
+  auto &http_server_instance_st{HttpServerInstanceState::get()};
```

Comment:
Just a thought: isn't it better to reduce number of variables in functions? Here, for example, we can get rid of `http_server_instance_st` as it's used only once

Reply:
fix

Reply:
fix